Userland/Libraries/LibWeb/HTML/Canvas/CanvasPath.cpp

```diff
@@ -120,6 +120,80 @@ WebIDL::ExceptionOr<void> CanvasPath::ellipse(float x, float y, float radius_x,
     return {};
 }
 
+// https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-arcto
+WebIDL::ExceptionOr<void> CanvasPath::arc_to(double x1, double y1, double x2, double y2, double radius)
+{
+    // 1. If any of the arguments are infinite or NaN, then return.
+    if (isnan(x1) || isnan(y1) || isnan(x2) || isnan(y2) || isnan(y2) || isnan(radius))
+        return {};
+    if (isinf(x1) || isinf(y1) || isinf(x2) || isinf(y2) || isinf(radius))
+        return {};
```

Comment:
I think we can combine the checks for infinite or NaN into `!isfinite(foo)`

Reply:
Thank you for the suggestion! I've made the update.